luwak/src/main/java/uk/co/flax/luwak/queryparsers/StandardLuceneQueryParser.java

```diff
@@ -0,0 +1,92 @@
+package uk.co.flax.luwak.queryparsers;
+
+/**
+ * Copyright (c) 2013 Lemur Consulting Ltd.
+ * <p/>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p/>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p/>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import java.util.HashMap;
+
+
+import java.util.Map;
+
+import org.apache.lucene.analysis.Analyzer;
+import org.apache.lucene.analysis.standard.StandardAnalyzer;
+import org.apache.lucene.queryparser.flexible.standard.StandardQueryParser;
+import org.apache.lucene.queryparser.flexible.standard.config.PointsConfig;
+import org.apache.lucene.search.Query;
+import uk.co.flax.luwak.MonitorQueryParser;
+
+/**
+ * A query parser that uses the default Lucene parser
+ */
+public class StandardLuceneQueryParser implements MonitorQueryParser {
```

Comment:
Could we change this to 'FlexibleLuceneQueryParser', as it's using the flex parser under the hood?